### PR TITLE
update project list and status

### PR DESCRIPTION
**Pull request by ritazh**
Signed-off-by: Rita Zhang <rita.z.zhang@gmail.com>

### CODE REVIEW

**Comment by ritazh** [comment]
@craiglpeters @sozercan @devigned @michelleN @aramase PTAL

**Comment by ritazh** [comment]
@aramase Let's add the kms plugin after the perf issue is sorted out.

**Comment by devigned** [comment]
What level of contribution are we setting as the min? There are quite a few that we have occasional contribution, but are not regular contributors / maintainers. The list seems to reflect the latter rather than the former.

**Comment by ritazh** [comment]
@devigned This list should be projects we maintain and support.

File: README.md [hunk 1]
```diff
@@ -24,24 +24,17 @@ Projects listed on this page are open source that Microsoft maintain or contribu
...
+|  | [IPv4/v6 Dual Stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) <br> ([kubernetes releases](https://github.com/kubernetes/kubernetes/releases)) <br> [Tests](https://testgrid.k8s.io/provider-azure-dualstack) | IPv4/IPv6 dual-stack enables the allocation of both IPv4 and IPv6 addresses to Pods and Services. | Kubernetes: <br> IPv6: beta <br> Dualstack: beta | [#sig-network](https://kubernetes.slack.com/archives/C09QYUH5W) <br> [kubernetes-sig-network@googlegroups.com](https://groups.google.com/forum/#!forum/kubernetes-sig-network) <br> [GitHub issues](https://github.com/kubernetes/kubernetes/labels/area%2Fipv6) | [Use with AKS Engine](https://github.com/Azure/aks-engine/tree/master/examples/dualstack) |
```
Comment (aramase) [inline README.md:30]:
```suggestion
|  | [IPv4/v6 Dual-Stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) <br> ([kubernetes releases](https://github.com/kubernetes/kubernetes/releases)) <br> [Tests](https://testgrid.k8s.io/provider-azure-dualstack) | IPv4/IPv6 dual-stack enables the allocation of both IPv4 and IPv6 addresses to Pods and Services. | Kubernetes: <br> IPv6: beta <br> Dual-stack: beta | [#sig-network](https://kubernetes.slack.com/archives/C09QYUH5W) <br> [kubernetes-sig-network@googlegroups.com](https://groups.google.com/forum/#!forum/kubernetes-sig-network) <br> [GitHub issues](https://github.com/kubernetes/kubernetes/labels/area%2Fipv6) | [Use dual-stack with AKS Engine](https://github.com/Azure/aks-engine/tree/master/examples/dualstack) <br> [Use IPv6 with AKS Engine](https://github.com/Azure/aks-engine/tree/master/examples/ipv6) |
```

Aligning the name with what we updated to in k/k

**Review by aramase — COMMENTED** [review]
Just one small comment, otherwise lgtm!

File: README.md [hunk 2]
```diff
@@ -24,24 +24,17 @@ Projects listed on this page are open source that Microsoft maintain or contribu
...
 | Cloud Native Service Mesh |  |  |  |  |
-|  | [Service Mesh Interface (SMI) Spec](https://smi-spec.io/) | A standard interface for service meshes on Kubernetes | CNCF: sandbox <br> APIs: Traffic Split v1alpha3; Traffic Target v1alpha1; Traffic Spec v1alpha2; Traffic Metrics v1alpha1 | [#smi](https://cloud-native.slack.com/messages/smi) <br> [GitHub issues](https://github.com/deislabs/smi-spec/issues) |  |
+|  | [Service Mesh Interface (SMI) Spec](https://smi-spec.io/) | A standard interface for service meshes on Kubernetes | CNCF: sandbox <br> APIs: Traffic Access Control: v1alpha3; Traffic Metrics: v1alpha1; Traffic Specs: v1alpha4; Traffic Split: v1alpha4  | [#smi](https://cloud-native.slack.com/messages/smi) <br> [GitHub issues](https://github.com/deislabs/smi-spec/issues) |  |
```
Comment (sozercan) [inline README.md:37]: can we add bullet points to APIs?

Comment (ritazh) [reply README.md:37]: @sozercan I tried bullet points before. it will make the height of that row really big such that the table looks terrible.

**Review by aramase — APPROVED** [review]

**Comment by sozercan** [comment]
we can add AKS to containerd also

File: README.md [hunk 3]
```diff
@@ -22,30 +22,23 @@ Projects listed on this page are open source that Microsoft maintain or contribu
...
+|  | [AKS Engine](https://github.com/Azure/aks-engine) <br> ([releases](https://github.com/Azure/aks-engine/releases)) | Self-managed clusters on Azure | Azure: incubating <br> API: N/A | [#aks-engine-users](https://kubernetes.slack.com/archives/CU3N85WJK) <br> [GitHub issues](https://github.com/Azure/aks-engine/issues) | <li>[Azure Stack Hub](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-kubernetes-aks-engine-overview)</li> |
+|  | [Cluster API Azure Provider](https://sigs.k8s.io/cluster-api-provider-azure) <br> ([releases](https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases)) <br> [Tests](https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-azure) | Self-managed clusters on Azure using Cluster API | CNCF: incubating <br> API: v1alpha4 | [#cluster-api-azure](https://kubernetes.slack.com/archives/CEX9HENG7) <br> [kubernetes-sig-cluster-lifecycle@googlegroups.com](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) <br> [GitHub issues](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues) |  |
```
Comment (devigned) [inline README.md:26]: Perhaps, switch the order to indicate suggested usage?

**Review by devigned — CHANGES_REQUESTED** [review]
Might be a good opportunity to put CAPZ first to indicate suggested usage and call out that AKS Engine is actually winding down.

**Review by devigned — APPROVED** [review]
/lgtm

**Review by sozercan — APPROVED** [review]

**Pull request merged**